src/shared/components/layouts/styles/side-bar-layout.scss

```diff
@@ -37,10 +33,21 @@
 
   &__sidebar {
     @include scroll-shadow;
```

Comment:
do we want to add that to `&__content` too?

Reply:
Can't really do that as we have 3 different sticky elements (page header, action buttons, table header) that would hide the shadow anyway.

Reply:
Maybe I was referring to the wrong thing. What is the bit which adds the shadow at the bottom of the sidebar when the footer is in view?

Reply:
This only adds a shadow to any scrollable area. The scrollable area here is the whole page.